src/main/java/com/wcc/platform/repository/postgres/PostgresMemberMemberTypeRepository.java

```diff
@@ -0,0 +1,46 @@
+package com.wcc.platform.repository.postgres;
+
+import com.wcc.platform.domain.platform.MemberType;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+/** Member to member type mapping repository */
+@Repository
+@RequiredArgsConstructor
+public class PostgresMemberMemberTypeRepository {
```

Comment:
We should check to use the default implementation with JPA with postgres, because it will help us to simplify a lot the repository